tests/test.cpp

```diff
@@ -15,6 +15,12 @@
  */
 #include <cmath>
 
+#ifdef FLATBUFFERS_GENERAL_HEAP_ALLOC_OK
```

Comment:
why is this re-def needed? tests should really always be able to use heap allocs

Reply:
Its redefining it to be True, overriding whatever was set in the cmake command. Its in the .cpp file, so it should be safe to redefine it in this case like this.

Reply:
ah ok.

Reply:
Its ok, I just removed it.